obspy/io/reftek/core.py

```diff
@@ -241,10 +241,14 @@ def to_stream(self, network="", location="", component_codes=None,
             else:
                 eh = EHPacket(et_packets[0])
             # only "C0" encoding supported right now
-            if eh.data_format != b"C0":
+            if eh.data_format == b"C0":
+                encoding = 'C0'
+            elif eh.data_format == b"C2":
+                encoding = 'C2'
```

Comment:
Hmm...good point. If `data_format` is not used for anything else I would vote for directly decoding it upon reading in accordance with the sandwich principle. But its your call and really a very minor detail.